### DISCUSSION TITLE
Support for Windows on ARM

### DISCUSSION THREAD

**Discussion by AChekroun** [OpenSC/OpenSC#3235] [Q&A] [answered]
Hello,

I'd like to acquire a Windows laptops based on one of latest Qualcomm's ARM chip.

Do you have any plan to support this architecture on Windows ?

Thanks

**Comment by tonyecj** [comment]
Same question.  Doesnt read card on windows Surface ARM laptop

**Comment by Jakuje** [comment, marked as answer]
Right now, thanks to @metsma we have CI that builds the OpenSC on ARM

You can find build artifacts in the GH Actions (this one is from the latest master build):

https://github.com/OpenSC/OpenSC/actions/runs/16455733577

Currently, we do not have arm builders in AppVeryor, which we use as a source for artifacts that go through code signing and to the releases, but we can adjust if there would be a demand.

**Reply by metsma** [reply]
There is one followup PR still open and some issues need still reaolved that are described there https://github.com/OpenSC/OpenSC/pull/3442. Right now only one set minidriver can be available. Arm64 or amd64 and it needs to match with running application arch
